tests/unit/actions.spec.ts

```diff
@@ -0,0 +1,118 @@
+import LegalServices from '@/services/legal-services'
```

Comment:
Please capitalize this file's name (consistent with class it's testing).

Reply:
Wait -- I see this is testing store actions, not Actions.vue.

How about renaming it "store-actions" ?

Reply:
Will do!